dev-tex/classico/classico-20140828.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit latex-package
+
+DESCRIPTION="LaTeX, pdfLaTeX, XeLaTeX and LuaLaTeX support for four URW Classico fonts"
+HOMEPAGE="https://www.ctan.org/tex-archive/fonts/urw/classico"
+SRC_URI="http://mirrors.ctan.org/install/fonts/urw/classico.tds.zip"
+
+LICENSE="free-noncomm"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~amd64-fbsd ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~x64-solaris ~x86-solaris"
```

Comment:
I don't think you should add all those arches you've probably never ever seen here.